partials/layout-head.htm

```diff
@@ -25,6 +25,5 @@
 <![endif]-->
 
 <script type="text/javascript" src="{{ 'javascript/combined.min.js'|theme_resource }}"></script>
-<script type="text/javascript" src="{{ 'javascript/combined-source/jquery.sharrre-1.3.4.js'|theme_resource }}"></script>
```

Comment:
The commit didn't have a note there (it should), so I didn't see it. Commits + commit messages should include real explanations (and real sentences), based on what's being checked in.